views/result_selection.py

```diff
@@ -95,6 +114,9 @@ def select_ts_datasource(self):
         return False
 
     def remove_selected_ts_ds(self):
+        """
+        remove selected result files from model (called by 'remove' button)
```

Comment:
Zoiets als "called by 'remove' button" is *erg* handig om erbij te zetten bij dit soort UI methodes.
(=Dat mag vaker als het elders er nog niet bij staat :-) )